gooddata-cn-on-eks/addons.tf

```diff
@@ -0,0 +1,152 @@
+locals {
+  kube_prometheus_stack = {
+    chart_version = helm_release.kps_crds.version
+    set_sensitive = [{
+      name  = "grafana.adminPassword"
+      value = var.grafana_password
+    }]
+    values = [
+      templatefile("${path.module}/files/kps-values.tftpl", {
+        dns_domain = data.aws_route53_zone.selected.name,
+      })
+    ]
+  }
+
+  ingress_nginx = {
+    values = [
+      templatefile("${path.module}/files/ingress-values.tftpl", {
+        lb_name         = local.name,
+        eip_allocations = join(",", aws_eip.lb[*].allocation_id),
+        dns_domain      = data.aws_route53_zone.selected.name,
+        cert_arn        = aws_acm_certificate.wildcard.arn,
+      })
+    ]
+  }
+}
+
+module "kube_prometheus_stack" {
+  source  = "aws-ia/eks-blueprints-addon/aws"
+  version = "1.1.1"
+
+  create = var.enable_kube_prometheus_stack
+
+  # Disable helm release
+  create_release = var.enable_kube_prometheus_stack
+
+  # https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/Chart.yaml
+  name             = try(local.kube_prometheus_stack.name, "kube-prometheus-stack")
+  description      = try(local.kube_prometheus_stack.description, "A Helm chart to install the Kube Prometheus Stack")
+  namespace        = try(local.kube_prometheus_stack.namespace, "kube-prometheus-stack")
+  create_namespace = try(local.kube_prometheus_stack.create_namespace, true)
+  chart            = try(local.kube_prometheus_stack.chart, "kube-prometheus-stack")
+  chart_version    = try(local.kube_prometheus_stack.chart_version, "48.2.3")
```

Comment:
this should be kept in sync with local.kps_version in gooddata-cn-on-eks/main.tf